go.mod

```diff
@@ -148,6 +148,7 @@ require (
 	github.com/spf13/viper v1.16.0 // indirect
 	github.com/stoewer/go-strcase v1.3.0 // indirect
 	github.com/subosito/gotenv v1.4.2 // indirect
+	github.com/test-network-function/test-network-function-claim v1.0.31 // indirect
```

Comment:
This shouldn't be needed, as we're using the claim package from this repo. You probably need to do `go mod tidy`.